helm-swoop.el

```diff
@@ -524,7 +535,7 @@ If $linum is number, lines are separated by $linum"
     (when (or (and (and (featurep 'migemo) helm-migemo-mode)
                    (migemo-forward $regex nil t))
               (re-search-forward $regex nil t))
-      (helm-swoop-flash-word (match-beginning 0) (match-end 0))
+      (helm-swoop-do-flash-region (match-beginning 0) (match-end 0))
```

Comment:
```
(funcall helm-swoop-flash-region-function (match-beginning 0) (match-end 0))
```
instead of define `helm-swoop-do-flash-region`?

Reply:
Why not a small util function? At least it looks meaningful than direct calling `funcall`.

Reply:
Increasing small utility functions requires a lot of consideration.

For example, all functions declared with `defsubst` seem to be prefixed with `helm-swoop--`, but how about renaming them?
And In many cases, I think `do` is unnecessary.